pypeit/spectrographs/mdm_modspec.py

```diff
@@ -0,0 +1,317 @@
+"""
+Module for MDM/Modspec specific methods.
+
+.. include:: ../include/links.rst
+"""
+
+import array as arr
+import numpy as np
+
+from astropy.time import Time
+
+from pypeit import msgs
+from pypeit import telescopes
+from pypeit.core import framematch
+from pypeit.spectrographs import spectrograph
+from pypeit.core import parse
+from pypeit.images import detector_container
+
+class MDMModspecEchelleSpectrograph(spectrograph.Spectrograph):
+    """
+    Child to handle MDM Modspec Echelle instrument+detector
+    """
+    ndet = 1
+    name = 'mdm_modspec_echelle'
+
+    telescope = telescopes.HiltnerTelescopePar()
+
+    camera = 'Echelle'
+    header_name = 'Modspec'
+    pypeline = 'MultiSlit'
```

Comment:
Should this really be MultiSlit? Perhaps this should be using the Echelle pypeline?

Reply:
I thought the same at first, but it isn't an Echelle.
It is only named that! :)